package.json

```diff
@@ -20,7 +20,7 @@
     "prepublish": "npm run compile"
   },
   "peerDependencies": {
-    "node-sass": "^3.5.3"
+    "node-sass": "^4.0.0"
```

Comment:
Agreed, this should be `>=3.5.3`... Not sure that`<5.0.0` is necessary, we don't know what version will break compatibility (e.g. `3.5.3` did).